commands.js

```diff
@@ -1207,6 +1208,12 @@ exports.commands = {
 		let targetUsername = (targetUser ? targetUser.name : target);
 
 		let buffer = [];
+		if (targetId === 'constructor') {
+			buffer.push('constructor user auth:');
+			buffer.push('No global or room auth.');
+			return connection.popup(buffer.join('\n\n'));
+		}
+
```

Comment:
If you're going to do this much hacking you may as well let it return "nice
try" or something